.github/workflows/createNewVersion.yml

```diff
@@ -0,0 +1,124 @@
+name: Create a new version
+
+on:
+  workflow_dispatch:
+    inputs:
+      SEMVER_LEVEL:
+        description: One of {BUILD, PATCH, MINOR, MAJOR}
+        required: true
+
+jobs:
+  createNewVersion:
+    if: github.actor == 'OSBotify'
+    runs-on: macos-latest
+
+    outputs:
+      PULL_REQUEST_NUMBER: ${{ steps.createPullRequest.outputs.pull-request-number }}
+
+    steps:
+      # Version: 2.3.4
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+
+      - uses: softprops/turnstyle@8db075d65b19bf94e6e8687b504db69938dc3c65
+        with:
+          poll-interval-seconds: 10
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up git
+        run: |
+          git fetch
```

Comment:
NAB: Do you know if these are chained like `&&` in bash, in that the next command won't run unless the previous command exits with a return code of `0`?

Reply:
That's a good question ... I do not know the answer.